molecule/delegated/tests/docker/main.py

```diff
@@ -190,3 +194,27 @@ def test_dockerlogin(host):
 
     package = host.package("pass")
     assert package.is_installed
+
+
+@pytest.mark.parametrize(
+    "name,image,expected_output",
+    [("docker_test", "docker.io/hello-world:latest", "Hello from Docker!")],
+)
+def test_docker_installation(host, name, image, expected_output):
+    # Check Docker version
+    docker_version = host.run("docker --version")
+    assert docker_version.rc == 0
+    assert "docker version" in docker_version.stdout.lower()
+
+    # Note: On CentOS, sudo is typically required for Docker commands due to SELinux.
+
+    # Run a test container
+    with host.sudo():
```

Comment:
Would ``host.sudo(get_variable(host, "operator_user"))`` work too?

Reply:
It works.